discord/voice_client.py

```diff
@@ -569,7 +571,13 @@ def _encrypt_xsalsa20_poly1305_lite(self, header: bytes, data) -> bytes:
 
         return header + box.encrypt(bytes(data), bytes(nonce)).ciphertext + nonce[:4]
 
-    def play(self, source: AudioSource, *, after: Optional[Callable[[Optional[Exception]], Any]] = None) -> None:
+    def play(
+        self,
+        source: AudioSource,
+        *,
+        after: Optional[Callable[[Optional[Exception]], Any]] = None,
+        **encoder_kwargs: Unpack[EncoderKwargs],
```

Comment:
Can you manually unpack these kwargs instead of using this form? It prevents people from typoing.